workflow/util/util.go

```diff
@@ -1037,3 +1037,94 @@ func GetNodeType(tmpl *wfv1.Template) wfv1.NodeType {
 	}
 	return ""
 }
+
+// RemoveDuplicates removes duplicate strings from slice
+func RemoveDuplicates(strSlice []string) []string {
```

Comment:
please don't add extra functions to this already too large file, please add to a new file

Reply:
Moved to a new file